maventa/types/MaventaAction.ts

```diff
@@ -0,0 +1,9 @@
+interface MaventaAction {
+    type: string;
+    channel: string;
+    message: string | null;
+    key: string | null;
+    happened_at: string;
+}
+
+export { MaventaAction }
```

Comment:
You can just write `export interface ... {}` and remove this extra `export { ... }`